recipes/stx/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.8)
+project(PackageTest LANGUAGES CXX)
+
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+find_package(stx CONFIG REQUIRED)
+
+add_executable(example example.cpp)
+target_link_libraries(example PRIVATE stx)
```

```suggestion
target_link_libraries(example PRIVATE stx::stx)
```

Usually, I prefer to not use cmake_find_package_multi in test_package if upstream doesn't provide CMake config file, but why not.

---

CI didn't report any issue with test_package even if it it links to non existent target `stx`. It means that current cpp file doesn't really test symbols defined in the lib file.

---

`conan_basic_setup()` sets up library search paths with `link_directories`,
so `target_link_libraries` is succeeding that way.

I can confirm that locally, not linking anything will cause a linker error:
error LNK2001: unresolved external symbol "void __cdecl stx::v1::begin_panic...."

---

hack to avoid namespace 👀